djangoproject/scss/_style.scss

```
@@ -1229,11 +1237,15 @@ a.cta {
         font-style: normal;
     }
 
-    &:hover,
-    &:focus {
+    &:hover{
```

Missing a space before the bracket:

```suggestion
    &:hover {
```